services/backend/package.json

```diff
@@ -20,8 +20,8 @@
   "author": "I-Talent team",
   "license": "MIT",
   "dependencies": {
-    "@prisma/client": "2.5.0",
-    "axios": "^0.19.2",
+    "@prisma/client": "2.5.1",
+    "axios": "^0.20.0",
```

Comment:
Maybe we should do this upgrade in a separate PR in case it starts causing issues (it will be easier to isolate)